.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.16
+      - image: circleci/golang:1.18
```

Comment:
Am I looking at a wrong direction?
![image](https://user-images.githubusercontent.com/1274789/168787353-5e316679-a7cc-4a50-be2b-33b843ff9f3a.png)

Reply:
Ah that's because they've moved to a new registry name under `cimg`. Will fix this